generators/server/templates/pom.xml.ejs

```diff
@@ -86,12 +98,13 @@
         <profile.api-docs />
         <profile.tls />
 
-        <!-- Dependency versions -->
         <jhipster-dependencies.version><%= JHIPSTER_DEPENDENCIES_VERSION %></jhipster-dependencies.version>
+        <entando-bundle-bom.version><%= ENTANDO_BUNDLE_BOM_VERSION %></entando-bundle-bom.version>
```

Comment:
should we include this one of the JHipster one regarding to the choice made by the user (instead of putting both always)?